lib/tasks/incrementals.rake

```diff
@@ -8,29 +8,32 @@ namespace :incrementals do
   desc 'Adds to the index'
   task :import, [:period] do |_task, args|
     indexer = Traject::Indexer::MarcIndexer.new
-    indexer.logger.info "name=Sirsi Incremental message=Indexing operation beginning task=#{args[:period]} import progress=start"
+    indexer.logger.info 'name="Sirsi Incremental" '\
+                        'message="Indexing operation beginning" '\
+                        "task=\"#{args[:period]} import\" "\
+                        'progress=start'
     indexer.load_config_file('lib/traject/psulib_config.rb')
     target = Dir["#{ConfigSettings.symphony_data_path}#{args[:period]}_#{ENV['RUBY_ENVIRONMENT']}/*.mrc"]
 
     if target.empty?
-      indexer.logger.info 'name=Sirsi Incremental '\
-                          'message=Nothing to index '\
-                          "task=#{args[:period]} import "\
+      indexer.logger.info 'name="Sirsi Incremental" '\
+                          'message="Nothing to index" '\
+                          "task=\"#{args[:period]} import\" "\
                           'progress=done'
     end
 
-    indexer.logger.info 'name=Sirsi Incremental '\
-                        "message=Indexing #{target} "\
-                        "task=#{args[:period]} import "\
-                        'progress=in progress'
+    indexer.logger.info 'name="Sirsi Incremental" '\
+                        "message=\"Indexing #{target}\" "\
+                        "task=\"#{args[:period]} import\" "\
+                        'progress="in progress"'
```

Comment:
no need for double quotes here?

Reply:
line 28?

Reply:
yup there is double quotes here for `in progress` but for example not on line 14 for `start`

Reply:
yes because it's two words. splunk interprets spaces as the end of the key/value pair.

I think this PR will make splunk happier, but I'm not 100% sure. Hard to test in dev. I could deploy this branch and see how it does?